README.md

```diff
@@ -37,6 +40,8 @@ Below is a car engine audio clip, and two two versions of the same clip, each on
 
 #### Your Answer
 <!-- Write your answer here -->
+car engine eq1= Here the timing between attacks sounds slowed down and the decay after the attack is longer. The pitch of sound is in a lower octave or lower pitch as thw sound is muddier and lower pitched than the first. 
+car engine eq2=Here the attack and decay is pretty short on the first sound and shortened on the second sound. It might be sped up, but it could just be the effect from the pitch being higher. The ranger of sound is even higher than the original. Visually on this sound wavelength you see it changes the most as it looks very streched out. 
```

Comment:
Great. The _fundamental_ pitch of both sound clips are the same, in EQ1 the first one has removed high frequencies, EQ2 has removed low frequencies. It does sounds like one is higher pitched than the other.
We'll go over EQ in class tonight.